requirements.txt

```diff
@@ -4,418 +4,147 @@
+pycparser==2.18           # via cffi
+pygithub==1.44.1          # via -r requirements.in
+pygments==2.5.2           # via ipython
+pyjwt==1.5.2              # via djangorestframework-jwt, pygithub, social-auth-core
+pyopenssl==17.5.0         # via -r requirements.in
+pyparsing==2.4.7          # via httplib2
+python-dateutil==2.8.1    # via -r requirements.in, botocore, elasticsearch-dsl, faker
+python-rapidjson==0.9.1   # via -r requirements.in
+python3-openid==3.1.0     # via social-auth-core
+python3-saml==1.9.0       # via -r requirements.in
+pytube==9.5.2             # via -r requirements.in
+pytz==2018.9              # via celery, django, zeep
+pyyaml==5.1.2             # via -r requirements.in
+redis==3.2.1              # via -r requirements.in, django-redis, django-server-status
+requests-file==1.4.3      # via tldextract
+requests-oauthlib==0.8.0  # via social-auth-core
+requests-toolbelt==0.9.1  # via zeep
+requests==2.24.0          # via -r requirements.in, akismet, betamax, django-anymail, ocw-data-parser, prawcore, pygithub, requests-file, requests-oauthlib, requests-toolbelt, smart-open, social-auth-core, tika, tldextract, update-checker, zeep
+rsa==4.0                  # via google-auth
+s3transfer==0.3.3         # via boto3
+sentry-sdk==0.14.1        # via -r requirements.in
+six==1.15.0               # via cryptography, django-anymail, django-appconf, django-bitfield, django-compat, django-guardian, django-imagekit, django-server-status, elasticsearch-dsl, faker, google-api-python-client, google-auth, html5lib, isodate, pygithub, pyopenssl, python-dateutil, requests-file, social-auth-app-django, social-auth-core, tenacity, traitlets, zeep
+smart-open==1.8.0         # via ocw-data-parser
+social-auth-app-django==2.1.0  # via -r requirements.in
+social-auth-core==1.7.0   # via social-auth-app-django
+soupsieve==2.0            # via beautifulsoup4
+sqlparse==0.3.0           # via django
+static3==0.5.1            # via -r requirements.in, dj-static
+tenacity==6.3.1           # via pyathena
+text-unidecode==1.2       # via faker
+tika==1.23.1              # via -r requirements.in
+tinycss2==0.6.1           # via cairosvg, cssselect2
+tldextract==2.2.0         # via -r requirements.in
+toolz==0.10.0             # via -r requirements.in
+traitlets==4.3.3          # via ipython
+ulid-py==0.0.3            # via -r requirements.in
+update-checker==0.16      # via praw
+uritemplate==3.0.0        # via google-api-python-client
+urllib3==1.25.10          # via -r requirements.in, botocore, elasticsearch, requests, sentry-sdk
+uwsgi==2.0.18             # via -r requirements.in
+vine==1.3.0               # via amqp, celery
+wcwidth==0.1.8            # via prompt-toolkit
+webencodings==0.5.1       # via html5lib, tinycss2
+wrapt==1.10.11            # via -r requirements.in, deprecated
+xbundle==0.3.1            # via -r requirements.in
+xmlsec==1.3.3             # via python3-saml
+zeep==3.4.0               # via mit-moira
```

Comment:
small thing: is there anyway to get pip-compile to maintain the line breaks, so this diff isn't so huge?

Reply:
fixed